corehq/apps/export/templates/export/partial/export_download_progress.html

```diff
@@ -53,19 +53,18 @@
                                 <span ng-show="!isMultimediaDownload">{% trans "Collecting Data" %}</span>
                                 <span ng-show="isMultimediaDownload">{% trans "Collecting Multimedia" %}</span>
                         </button>
-                        {% if request|toggle_enabled:"EMAIL_EXPORT_WHEN_DONE_BUTTON" %}
                         <button type="button"
-                                title="{% trans 'For exports that may take a while to prepare' %}"
+                                title="{% trans 'For exports that may take a while to prepare, email a link when download is ready' %}"
+                                style="margin-top: 5px;"
                                 ng-show="!isDownloadReady"
                                 ng-init="sendEmailFlag = false;"
                                 ng-class="'btn btn-' + (sendEmailFlag ? 'default' : 'primary') + ' btn-full-width'"
                                 ng-disabled="sendEmailFlag"
                                 ng-click="sendEmailUponCompletion(); sendEmailFlag = true;">
                                 <i class="fa fa-envelope"></i>
-                                {% trans "Email me the download link when ready" %}
+                                {% trans "Email me when ready" %}
```

Comment:
There had been some UAT feedback that people may be uncomfortable receiving "it" in their email because of HIPAA concerns, so that text was added to make it clear that what they receive is a _link_ to download the file, not the file itself as an attachment.

May be worth adding that in a different way somehow as help text or something :man_shrugging:

Reply:
When you hover on the button it says that a link will be emailed. I could put that info in a little help block below the button if that'd be better, what do you think?

Reply:
(I lean towards the hover being okay, but don't feel strongly about it.)

Reply:
Oops, I see. I'd missed that when I made my last comment. Maybe for now let's say the hover is plenty; I could see people still missing that (I bet lots of people don't think to mouseover), but we can wait for that feedback.

Reply:
sounds good